pages/common/ical.md

```diff
@@ -0,0 +1,15 @@
+# ical
+
+> Program to display english Calendar on your terminal
```

Comment:
```suggestion
> Program to display english Calendar on your terminal.
```
According to <https://manned.org/ical>, `ical` converts the given Gregorian date to a Hirji calendar.

Reply:
>and vice-versa